packages/nextjs/app/builders/0x6d3321b4637D8451740Cf400848602f838F7A594/page.tsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import { FaGithub } from "react-icons/fa";
+import { SiGmail } from "react-icons/si";
+import { Address } from "~~/components/scaffold-eth";
+import { BlockieAvatar } from "~~/components/scaffold-eth";
+
+const PersonalPage = () => {
```

Comment:
You might want to consider typing your page as NextPage since we are using TypeScript, as this ensures better integration and adds clarity for future maintainers by indicating that the component is a Next.js page.

Also you could consider it after your name since there are are many more personal pages.